samples/nextjs/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Can we make this more strict to make sure we have a great sample app? We should add
noImplicitAny
noImplicitReturns
noImplicitThis
noUnusedLocals
noUnusedParameters